### DISCUSSION TITLE
Group sequential photos

### DISCUSSION THREAD

**Discussion by webysther** [photoprism/photoprism#2646] [Ideas]
Will be great to make possible group timelapse sequential photos:

![image](https://user-images.githubusercontent.com/750007/186686803-893928a1-784b-4201-9248-3f98fd435dcf.png)

**Comment by graciousgrey** [comment]
Timelapse photos can already be stacked in case pictures were taken at the same GPS position and second (optional)

see https://docs.photoprism.app/user-guide/organize/stacks/

Related open issue:
https://github.com/photoprism/photoprism/issues/1446

**Reply by webysther** [reply]
**Time**-**lapse** don't never will be in the same second, look, if this is a option, this is for the all library, will be great select the photos to be grouped. :)

**Reply by graciousgrey** [reply]
Manual stacking is planned as well https://github.com/photoprism/photoprism/issues/28 :)
